specutils/fitting/fitmodels.py

```diff
@@ -37,6 +42,19 @@ def fit_lines(spectrum, model, fitter=fitting.LevMarLSQFitter(),
         Regions of the spectrum to use in the fitting. If None, then the
         whole spectrum will be used in the fitting.
 
+    maxiter : int
```

Comment:
Remove these three docstring entries and instead say 
```
additional keyword arguments are passed directly into the call to the ``fitter``
```
since there's no guarantees these specific arguments are available on the fitters now.